src/commands/login.rs

```diff
@@ -58,9 +63,17 @@ pub fn execute<'a>(_matches: &ArgMatches<'a>) -> Result<(), Error> {
         }
     }
 
-    update_config(&config, &token)?;
-    println!();
-    println!("Stored token in {}", config.get_filename().display());
+    let mut config_to_update = local_config;
+    if matches.is_present("global") {
+        let (global_filename, global_config) = load_global_config_file()?;
+        config_to_update = Config::from_file(global_filename, global_config)?.bind_to_process();
+    }
+
+    update_config(&config_to_update, &token)?;
+    println!(
+        "\nStored token in {}",
```

Comment:
_nit_: Rather than leading with a newline, could you add a `println!();` above?

Reply:
It was like this before and I changed it because it looked unnecessary. Will change it back.